modules/storage-persistent-storage-pv.adoc

```diff
@@ -249,12 +256,14 @@ cluster.
 
 |===
 
-You can view the name of the PVC that is bound to the PV by running the following command:
```

Comment:
why did you remove this one?

Reply:
It's not entirely accurate which might be the reason, if we want to keep that it could use some refinement. It says "view name of PVC that is bound to the PV" but `oc get pv <pv-name>`  gets you the whole object. We can leave this text and change command to: `oc get pv <pv-name> -o jsonpath='{.spec.claimRef.name}'`

Reply:
I didn't realy remove this line, but rather moved it higher up in the topic. But I have corrected the command to Roman's suggestion.